sdk/core/core/src/az_http_response.c

```diff
@@ -216,6 +219,9 @@ az_http_response_get_next_header(az_http_response* response, az_pair* out_header
     out_header->value = az_span_slice(*reader, 0, offset_value_end);
     // moving reader. It is currently after \r was found
     *reader = az_span_slice_to_end(*reader, offset);
+
+    // Remove white spaces from header name https://github.com/Azure/azure-sdk-for-c/issues/604
```

Comment:
Comment/code mismatch?
```suggestion
    // Remove white spaces from value https://github.com/Azure/azure-sdk-for-c/issues/604
```